src/assignment/views/AssignmentCreate.tsx

```diff
@@ -0,0 +1,809 @@
+									onChange={value => setAssignmentProp('answer_url', value)}
+								/>
+								<p className="c-form-help-text">
+									<Trans i18nKey="assignment/views/assignment-edit___waar-geeft-de-leerling-de-antwoorden-in-voeg-een-optionele-url-naar-een-ander-platform-toe">
+										Waar geeft de leerling de antwoorden in? Voeg een optionele URL naar een ander
+										platform toe.
+									</Trans>
+								</p>
+							</FormGroup>
+							<FormGroup label={t('assignment/views/assignment-edit___beschikbaar-vanaf')}>
+								<Flex>
+									<DatePicker
+										value={
+											currentAssignment.available_at
+												? new Date(currentAssignment.available_at)
+												: now
+										}
+										onChange={(value: Date | null) =>
+											setAssignmentProp('available_at', value ? value.toISOString() : null)
+										}
+										showTimeInput
+									/>
+								</Flex>
+							</FormGroup>
+							<FormGroup label={t('assignment/views/assignment-edit___deadline')} required>
+								<Flex>
+									<Spacer margin="right-small">
+										<DatePicker
+											value={
+												currentAssignment.deadline_at
+													? new Date(currentAssignment.deadline_at)
+													: null
+											}
+											onChange={value => setAssignmentProp('deadline_at', value)}
+											showTimeInput
+										/>
+									</Spacer>
+								</Flex>
+								{isDeadlineInThePast() ? (
+									<div className="c-form-help-text c-form-help-text--error">
+										<Trans i18nKey="assignment/views/assignment-edit___de-deadline-ligt-in-het-verleden">
+											De deadline ligt in het verleden.
+										</Trans>
+										<br />
+										<Trans i18nKey="assignment/views/assignment-edit___de-leerlingen-zullen-dus-geen-toegang-hebben-tot-deze-opdracht">
+											De leerlingen zullen dus geen toegang hebben tot deze opdracht
+										</Trans>
+									</div>
+								) : (
+									<p className="c-form-help-text">
+										<Trans i18nKey="assignment/views/assignment-edit___na-deze-datum-kan-de-leerling-de-opdracht-niet-meer-invullen">
+											Na deze datum kan de leerling de opdracht niet meer invullen.
+										</Trans>
+									</p>
+								)}
+							</FormGroup>
+							{currentAssignment.assignment_type === 'BOUW' && (
+								<FormGroup
+									label={t('assignment/views/assignment-edit___groepswerk')}
+									labelFor="only_player"
+								>
+									<Toggle
+										checked={currentAssignment.is_collaborative}
+										onChange={checked => setAssignmentProp('is_collaborative', checked)}
+									/>
+								</FormGroup>
+							)}
+							<hr className="c-hr" />
+							<Alert type="info">
+								<div className="c-content c-content--no-m">
+									<p>
+										<Trans i18nKey="assignment/views/assignment-edit___hulp-nodig-bij-het-maken-van-opdrachten">
+											Hulp nodig bij het maken van opdrachten?
+										</Trans>
+										<br />
+										Bekijk onze{' '}
```

Comment:
Moet dit ook niet translatable zijn in de toekomst?

Reply:
ja idd, zal ik in de toekomst nog aanpassen in een aparte PR

Reply:
https://district01.atlassian.net/browse/AVO2-1006